specification/electronic-prescription-service-api.yaml

```diff
@@ -320,7 +282,13 @@ paths:
                   value:
                     $ref: components/examples/secondary-care/community/acute/nominated-pharmacy/nurse/prescribing-and-professional-codes/1-Process-Response-Send-200_OK.json
         '4XX':
-          description: Invalid request.
+          description: |
+            An error occurred as follows:
+            
+            | HTTP status | Error code         | Description                                    | FHIR type        |
+            | ----------- | ------------------ | ---------------------------------------------- | -----------------|
+            | 400         | BAD_REQUEST        | Spine Error - duplicate prescription ID exists | OperationOutcome |
```

Comment:
Don't really understand the `FHIR type` bit of this. What's it telling me?

Reply:
supposed to explain to a user that some errors will be OperationOutcomes and some Bundles - this hopefully is temporary until we clean up all errors being OperationOutcomes

Reply:
Also does this section override the `API specific errors` bit above or add to it? Because we've defined `400` responses in that section

Reply:
Ah okay, we could use the examples or schema to show that there are multiple possible error formats, but this is fine for now

Reply:
yea i agree with this, might merge in and suggest to sachin to have a look into that :+1: